src/components/Organisms/Donate/MoneyBox/MoneyBox.js

```diff
@@ -50,23 +54,27 @@ const MoneyBuy = ({
   currency,
   description,
   ...rest
-}) => (
-  <MoneyBox
-    {...rest}
-    aria-label={description}
-    value={`${currency} ${amount}`}
-    type="button"
-    label=""
-    errorMsg=""
-    onClick={setOtherAmount}
-  />
-);
+}) => {
+  // Set decimal points accordingly
+  const formattedAmount = amountFormatter(amount);
+  return (
+    <MoneyBox
+      {...rest}
+      aria-label={description}
+      value={`${currency} ${formattedAmount}`}
```

Comment:
More of a query than anything else. This component accepts a currency, and an amount, but we just pass both as single string. I just wondered why this was? Potentially again, outside of the scope of this work but just something I noticed and thought I would query.

Reply:
Ehhh, there's definitely a lot to be refactored in this old (not my) work; I guess there's cause to keep both those variables separate to do _number_-specific things to `amount`, but we need the currency symbol here for aesthetics purely?

Not sure if I misunderstood you here?

Reply:
That makes sense, it works fine this way, was just curious as it its purpose. 😄